### PR TITLE
chore: merge code from appium-gui-libs submodule

### PR DESCRIPTION
**Pull request by eglitise**
This PR fully migrates the contents of the appium-gui-libs submodule into this repo.

Some files were moved directly, others were integrated with existing files. `menus.js` was refactored the most, since it had a lot of identical imports being passed around.

The app seems to work fine, hopefully nothing has broken during the migration 🤞

### CODE REVIEW

**Review by KazuCocoa — COMMENTED** [review]
Thank you!.
looks good to me, overall. I'll build this branch and check a few

File: app/main/auto-updater/update-checker.js [hunk 1]
```diff
@@ -19,4 +19,113 @@ export async function checkUpdate (currentVersion) {
...
+      return;
+    }
+    const update = await checkUpdate(app.getVersion());
+    if (update) {
+      let {name, notes, pub_date: pubDate} = update;
+      pubDate = moment(pubDate).format(i18n.t('datetimeFormat'));
+
+      let detail = i18n.t('updateDetails', {pubDate, notes: notes.replace('*', '\n*')});
+      if (env.NO_AUTO_UPDATE) {
+        detail += `\n\nhttps://www.github.com/appium/appium-desktop/releases/latest`;
```
Comment (jlipps) [inline app/main/auto-updater/update-checker.js:52]: at this point we should probably change this to `appium-inspector`

**Review by jlipps — APPROVED** [review]
incredible job @eglitise! thank you

with all of your recent contributions i'd like to invite you to be maintainer of this project. no obligation for further work of course but if you want you will be able to help review and merge PRs.

many people around the world use this tool and we need all the help we can get to keep it running well and move forward with exciting new features. i know that many people have already appreciated your contributions!

File: app/main/auto-updater/update-checker.js (hunk 1 above)
Comment (eglitise) [reply app/main/auto-updater/update-checker.js:52]: I agree, but considering that this isn't the only remaining reference to Appium Desktop, I was thinking to make a separate PR for fixing all of them.

**Review by KazuCocoa — APPROVED** [review]
lgtm

**Pull request merged**